README.md

```diff
@@ -1,5 +1,4 @@
-Yoyodyne 🪀
-==========
+# Yoyodyne 🪀
```

Comment:
Are these formatting changes intentional? Have I or someone else been reverting them and you keep having to change them back? I feel like I often see your PRs update the README markdown format in this same way.

Reply:
I use Pandoc (from markdown to markdown) to reflow the README when I make major changes to the README.